docs/get-started/_index.md

```diff
@@ -112,14 +99,40 @@ Viam uses WebRTC and gRPC for secure peer-to-peer communications across network
 If `viam-server` can connect to the internet, you can control and monitor your machines securely **from anywhere in the world**.
 In scenarios where machines are connected to a local area network (LAN) with intermittent or no cloud connectivity, you can control and monitor machines from the local network or with code on the machines themselves.
 
+### Support
+
+The `viam-server` executable runs on **any** computer that runs on 64-bit Linux OS, macOS Computer, or Windows Subsystem for Linux.
+Here are some of the boards Viam supports:
+
+{{< board-carousel >}}
+<br>
+
+Viam also supports microcontrollers with the [micro-RDK](/installation/#install-micro-rdk).
+
+## Viam Cloud
+
+The [Viam app](https://app.viam.com) provides the user interface for configuring your machine, a registry of resources for your machine, as well as capabilities to make your machines better and smarter.
+
+### Configuration
+
+As you configure your machine, you can test each resource to confirm it is working as expected in the same UI you use for configuration.
+
+{{<gif webm_src="/test.webm" mp4_src="/test.mp4" alt="Test a camera stream" max-width="600px" class="aligncenter">}}
+
+### Extensibility
+
+For everything that is not built-in, you can deploy _{{< glossary_tooltip term_id="module" text="modules" >}}_ from the [**Viam Registry**](/registry/).
+Modules provide custom components or services as _modular resources_ which are run and managed by `viam-server` as processes.
```

Comment:
```suggestion
Modules provide custom components or services as _modular resources_ which are run and managed by `viam-server` as processes.
```
should we link out to processes here or is that confusing?

Reply:
not that kind of processes, they're run as system processes. Not the processes you configure in the configuration of a machine